src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1284,6 +1315,51 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
             return default(ImmutableArray<RefKind>);
         }
 
+        private delegate BoundExpression ParamsArrayElementRewriter<TArg>(BoundExpression element, ref TArg arg);
```

Comment:
Consider using `in` for `arg` instead.

Reply:
> Consider using `in` for arg instead.

`Ref` is intentional, the argument is not read-only.